plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/error_resolution/AbstractFBErrorMarkerResolution.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Primetals Technologies Austria GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Sebastian Hollersbacher
+ *    - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+package org.eclipse.fordiac.ide.application.error_resolution;
```

Comment:
Please follow the Eclipse Naming Convention for package names, particularly https://wiki.eclipse.org/Naming_Conventions#Additional_rules:
> Package names should contain only lowercase ASCII alphanumerics, and **avoid underscore** _ or dollar sign $ characters.

```suggestion
package org.eclipse.fordiac.ide.application.marker.resolution;
```